includes/class-edd-license-handler.php

```diff
@@ -120,6 +118,8 @@ private function hooks() {
 
 		add_action( 'in_plugin_update_message-' . plugin_basename( $this->file ), array( $this, 'plugin_row_license_missing' ), 10, 2 );
 
+		// Register plugins for beta support
+		add_filter( 'edd_has_beta_support', array( $this, 'register_beta_support' ) );
```

Comment:
Actually this is the same filter that exists in the proposed changes to `edd_has_beta_support()` function, I would think since this is an array of extensions that have support it should read something like `edd_beta_enabled_extensions`